{"uuid":"cfa14bf8-2f27-4dac-bc2b-fa62f10f14ec","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["3012f3a1-4bcf-45e4-a2c8-9b085437bc91"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879615300,"stop":1696879615301},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879615431,"stop":1696879615432},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879615588,"stop":1696879615588},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879617633,"stop":1696879617633}],"start":1696879614984,"stop":1696879617633}